usaspending_api/etl/es_mapper.json

```diff
@@ -0,0 +1,121 @@
+{
+  "settings": {
+    "index.mapping.ignore_malformed": true,
+    "index.max_result_window" : 500000
+  },
+  "sort": {
+    "fields": [
+      "transaction_amount",
+      "action_date"
+    ],
+    "order": [
+      "desc",
+      "desc"
+    ]
+  },
+  "mappings": {
+    "transaction_mapping": {
+      "properties": {
+        "awarding_subtier_agency_name": {
+          "fielddata": true,
+          "type": "text"
+        },
+        "awarding_toptier_agency_name": {
+          "type": "text",
+          "fielddata": true
+        },
+        "action_date": {
+          "type": "date",
+          "format": "yyyy-MM-dd"
+        },
+        "period_of_performance_start_date": {
+          "type": "date",
+          "format": "yyyy-MM-dd"
+        },
+        "period_of_performance_current_end_date": {
+          "type": "date",
+          "format": "yyyy-MM-dd"
+        },
+        "award_id": {
+          "type": "integer"
+        },
+        "transaction_id": {
+          "type": "integer"
+        },
+        "total_obligation": {
+          "type": "float"
+        },
+        "session_id": {
+          "type": "text",
+          "index": false
+        },
+        "transaction_amount": {
```

Comment:
`transaction_amount` was originally an alias of `federal_action_obligation`. Only need one of these